samples/sensor/grow_r502a/boards/yd_esp32.overlay

```diff
@@ -0,0 +1,28 @@
+&pinctrl {
+	uart2_default: uart2_default {
+		group1 {
+			pinmux = <UART2_TX_GPIO32>;
+		};
+		group2 {
+			pinmux = <UART2_RX_GPIO33>;
+			bias-pull-up;
+		};
+	};
+};
+
+&uart2 {
+	status = "okay";
+	current-speed = <57600>;
+	pinctrl-0 = <&uart2_default>;
+	pinctrl-names = "default";
+
+	fps {
+		#address-cells=<1>;
+		#size-cells=<0>;
+		grow_r502a@ffffffff {
```

Comment:
Does this board has `grow_r502a` integrated as well?  
@henrikbrixandersen @kartben Question: Does it make sense to add such overlay considering that the board does not have this sensor integrated? Well, I am ok with that but I am not sure if this is a standard procedure. Otherwise all boards would need overlay for all samples.